packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -20,6 +20,10 @@ export const Shadow = stagedComponent((props: ShadowProps) => {
       if (childrenArray.length !== 1) {
         console.warn('Shadow must only have one child');
       }
+
+      if (child.type != View) {
+        console.warn('Shadow child must be a view');
+      }
```

Comment:
Talked offline: As it turns out this won't catch things derived from View, like Pressable. So I think if you put a `Button` here, it would also fail :(

Reply:
Yep will remove the warning altogether. Will add jest tests too, we should have tests for Shadow on different types of components.

Reply:
Yeah I think having Shadows wrap a few different components is a good test / example we can add here

Reply:
Actually Button doesn't trigger this warning (at least FAB doesn't). Notification does though, so will add tests for both Button and Notification

Reply:
Removed the tests for Button and Notification because I think they add circular dependencies...should be getting tested in those respective component jest tests though.